tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+from utils import missing_feature, context, scenarios
+
+
+@scenarios.parametric
```

Comment:
I know we're dealing with OpenTelemetry...but all the other OTEL classes have this same features name 😆 

```suggestion
@scenarios.parametric
@features.open_tracing_api
```